deploy/arbitrum/261_depoy_fluidity_RewardScheduler.ts

```diff
@@ -0,0 +1,29 @@
+import { ethers } from "hardhat"
+import { DeployFunction } from "hardhat-deploy/types"
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { fUSDC_Reward_Manager_Address } from "../../utils/accounts"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre
+  const { deploy, get } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  const RewardScheduler = await deploy(
+    "RewardScheduler_ChildGauge_CommunityfUSDCLPToken",
+    {
+      from: deployer,
+      contract: "RewardScheduler",
+      log: true,
+      skipIfAlreadyDeployed: true,
+      args: [fUSDC_Reward_Manager_Address, (await get("fUSDC")).address],
```

Comment:
Looks like a wrong address used for RewardForwarder associated with the ChildGauge of CommunityfUSDCPoolLPToken. Should be the RewardForwarder.